spirv_reflect.c

```diff
@@ -5327,7 +5327,7 @@ const char* spvReflectSourceLanguage(SpvSourceLanguage source_lang)
     case SpvSourceLanguageSYCL           : return "SYCL";
     case SpvSourceLanguageHERO_C         : return "Hero C";
     case SpvSourceLanguageNZSL           : return "NZSL";
-    case SpvSourceLanguageMax:
+    default:
       break;
   }
   return "";
```

Comment:
```suggestion
  return "SpvSourceLanguage not found";
```

Reply:
I think the difficulty with this one is there exists a `SpvSourceLanguageUnknown` symbol - which currently returns the "Unknown".  My instinct is to say that the the case for that should be removed from the switch and and the default should break and "Unknown" should be returned. It looks like this is what `spirv-dis` does: https://github.com/KhronosGroup/SPIRV-Tools/blob/main/source/opcode.cpp#L64

Reply:
Done.